### DISCUSSION TITLE
HELP ME

### DISCUSSION THREAD

**Discussion by chifouu65** [Minestom/Minestom#1018] [Help] [answered]
hello  i have a problm when starting the server the minestorm package say "cannot acces package'name'"

**Comment by emortaldev** [comment]
The gradle script or some code would be nice

**Reply by chifouu65** [reply]
public static void main(String[] args) {
        MinecraftServer minecraftServer = MinecraftServer.init();
        InstanceManager instanceManager = MinecraftServer.getInstanceManager();
        // Create the instance
        InstanceContainer instanceContainer = instanceManager.createInstanceContainer();
        // Set the ChunkGenerator
        instanceContainer.setGenerator(unit ->
                unit.modifier().fillHeight(0, 40, Block.GRASS_BLOCK));
        // Add an event callback to specify the spawning instance (and the spawn position)
        GlobalEventHandler globalEventHandler = MinecraftServer.getGlobalEventHandler();
        globalEventHandler.addListener(PlayerLoginEvent.class, event -> {
            final Player player = event.getPlayer();
            event.setSpawningInstance(instanceContainer);
            player.setRespawnPoint(new Pos(0, 42, 0));
        });
        // Start the server on port 25565
        minecraftServer.start("0.0.0.0", 25565);
    }

**Reply by chifouu65** [reply]
i have code :/

**Reply by emortaldev** [reply]
What about the gradle script

**Reply by chifouu65** [reply]
i use maven

**Reply by emortaldev** [reply]
Ok I'm probably unable to help then, however posting the pom would be useful. I'd recommend editing the original question to include the pom so people can see it easier, instead of buried within replies.
I would recommend using Gradle if you are not already accustomed to Maven as it is what most people that use Minestom use, meaning you can get more support. Otherwise you may just want to look into "setting main class" or "fat jars" for Maven

**Reply by chifouu65** [reply]
can you send me a build.gradle pls ?

**Reply by emortaldev** [reply]
Check [the Minestom Wiki](https://wiki.minestom.net/setup/your-first-server), it covers a lot of topics for starting a new server

**Comment by chifouu65** [comment]
i have test on maven and i have any problm

**Comment by TheMode** [comment, marked as answer]
Due to the lack of information, the only thing I can say is look at the documentation https://wiki.minestom.net/setup/your-first-server

**Reply by chifouu65** [reply]
is jdk 16 for minestorm ?

**Reply by chifouu65** [reply]
minestom*

**Reply by emortaldev** [reply]
Minestom will use latest Java, it is currently 17 (also an edit feature exists)

**Comment by emortaldev** [comment]
I think the most simple answer is to check the [Minestom Wiki](https://wiki.minestom.net/setup/your-first-server), it covers just about everything you need in order to get started with your first Minestom sever using Gradle.
